dominion/dominion_ui/src/components/Hero/Hero.js

```diff
@@ -74,32 +103,48 @@ const styles = theme => ({
   }
 });
 
-function HeroTitleGridComponent({ classes, children }) {
+function HeroTitleGridComponent({ classes, children, quater }) {
   return (
-    <Grid item sm={12} md={8} lg={8} xl={6} className={classes.titleTextGrid}>
+    <Grid
+      item
+      xs={12}
+      sm={12}
+      md={quater ? 4 : 8}
+      lg={quater ? 4 : 8}
+      xl={quater ? 4 : 6}
+      className={classes.titleTextGrid}
+    >
       {children}
     </Grid>
   );
 }
 
 HeroTitleGridComponent.propTypes = {
   classes: PropTypes.isRequired,
-  children: PropTypes.isRequired
+  children: PropTypes.isRequired,
+  quater: PropTypes.bool.isRequired
 };
 
 const HeroTitleGrid = withStyles(styles)(HeroTitleGridComponent);
 
-function HeroTitleComponent({ classes, children }) {
+function HeroTitleComponent({ classes, children, breakWord, small }) {
   return (
-    <Typography variant="h1" className={classes.text}>
+    <Typography
+      variant="h1"
+      className={`${classes.title} ${
+        breakWord ? classes.titleWordBreak : null
+      } ${small ? classes.titleFontSmall : null}`}
+    >
```

Comment:
Looks like you want to have multiple classes here. If that's the case, we've already included [classnames](https://github.com/JedWatson/classnames), a small library to handle such cases elegantly.